packages/main/src/themes/Toast.css

```diff
@@ -0,0 +1,89 @@
+:host {
+	font-family: var(--sapUiFontFamily);
+	color: var(--sapUiListTextColor);
+	font-size: var(--sapMFontMediumSize);
+}
+
+.ui5-toast-root {
+	position: fixed;
+	left: 0;
+	right: 0;
+	top: 0;
+	bottom: 0;
+	width: 100%;
+	display: none;
+	height: 100%;
+	pointer-events: none;
+	overflow: hidden;
+	z-index: 999;
+	justify-content: center;
+	align-items: flex-end;
+}
+
+.ui5-toast-content {
+	box-sizing: border-box;
+	max-width: 15rem;
+	overflow: hidden;
+	padding: 1rem;
+	background: var(--_ui5_toast_background);
+	box-shadow: var(--_ui5_toast_shadow);
+	border-radius: .25rem;
+	transition-property: opacity;
+	opacity: 1;
+	margin: 2rem;
+}
+
+.ui5-toast-text {
```

Comment:
Did not test it, but if you move the styles from `.ui5-toast-text {}` selector to the `.ui5-toast-content{}` selector,
you might be able to skip the `<span class="ui5-toast-text"` in the HBS file and lightweight the DOM structure a bit.